Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/model/Settings.java

```diff
@@ -17,6 +17,9 @@
   public static final String SECTION_INI_CORE = "Core";
   public static final String SECTION_INI_INTERFACE = "Interface";
 
+	public static final String SECTION_LOGGER_LOGS = "Logs";
```

Comment:
Those appear to be tabs instead of two spaces. Thats why Lint is complaining.

Reply:
Android Studio's code reformatting missed that and didn't change the tabs to two spaces... Something's definitely wrong with Android Studio's conformance to code style or my setup.